src/proxy-ui-api/src/main/resources/openapi-definition.yaml

```diff
@@ -2,7 +2,7 @@
 openapi: 3.0.0
 info:
   description: X-Road Security Server Admin API. Note that the error metadata responses described in some endpoints are subjects to change and may be updated in upcoming versions.
-  version: "1.0.32"
+  version: "2.0.0"
```

Comment:
Incrementing the major version seems drastic compared to the level of changes

Reply:
It does a bit, yes. However it is in principle a breaking change. Discussion where use of 2.0.0 was decided: https://niis.slack.com/archives/GGBHUKL68/p1629096767038000?thread_ts=1628776238.030800&cid=GGBHUKL68